src/api/spec/models/cloud/upload_job_spec.rb

```diff
@@ -34,7 +34,10 @@
   end
 
   describe 'validations' do
-    it { is_expected.to validate_inclusion_of(:arch).in_array(['x86_64']) }
+    it do
```

Comment:
So we are testing that validations from [ActiveModel](https://api.rubyonrails.org/classes/ActiveModel/Validations/HelperMethods.html) are working? I don't see the benefits in having such tests.

Reply:
It's rather to verify that such validations exist, with the correct scope, etc.